testomania/src/main/java/com/earth/testomania/common/Util.kt

```diff
@@ -0,0 +1,20 @@
+package com.earth.testomania.common
+
+import android.util.Log
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+
+const val GENERIC_TAG = "TESTOMANIA_TAG"
+
+fun log(text: String) {
+    Log.d(GENERIC_TAG, text)
+}
+
+object Crashlytics {
```

Comment:
why not provide this via Hilt?

Reply:
Yes, I also thought this, but we need it not only in view model and i do not want to inject it in every place, cause writing this is so simple. So i think for now we do not need to inject it